Backendless/Backendless.cs

```diff
@@ -98,6 +99,26 @@ public static void InitApp( String applicationId, String apiKey )
       AppId = applicationId;
       APIKey = apiKey;
 
+      InitAppData = new InitAppData();
+      InitApp();
+    }
+    
+    public static void InitApp( String customDomain )
+    {
+      if( String.IsNullOrEmpty( customDomain ) )
+        throw new ArgumentNullException( "Custom domain cannot be null or empty" );
+
+      if( customDomain.StartsWith( "http" ) )

```

Comment:
bring customDomain to lower case before you do the check

Reply:
no, do this instead:
if( customDomain.ToLower().StartsWith( "http" ) )
if they provided the URL with some letters capitalized, we will use what they provided.